openbts/components.py

```diff
@@ -41,6 +42,37 @@ def monitor(self):
     }
     return self._send_and_receive(message)
 
+  def tmsis(self, fields=['IMSI', 'ACCESSED'], access_period=0, auth=2):
+    """Gets all active subscribers from the TMSI table.
+
+    Args: access_period: fetches all entries with ACCESS < acess_period
+                         (default=0 filter off)
+          auth: fetches all entries with AUTH = auth
+                         (default=2)
+
+    Returns: a list of objects defined by the list of fields.
```

Comment:
I feel like this API should help out the user more -- like there should be a consistent reply and you shouldn't have to specify fields.  Why not just return IMEI, IMSI, TMSI, CREATED, ACCESSED and other fields that you think we'll need in `client`?

I guess I'm suggesting you make this method more than a light wrapper around NM.  I think if this command does more processing / parsing, the people using it in `client` will have an easier time..